dss/events/handlers/sync.py

```diff
@@ -17,6 +18,10 @@
 from dss.util.aws import resources, clients, send_sns_msg, ARN
 from dss.util.streaming import get_pool_manager, S3SigningChunker
 
+
+logger = logging.getLogger(__name__)
+
```

Comment:
Nitpick: This is one blank line too many

Reply:
Will fix.